wd_helper.au3

```diff
@@ -2196,6 +2196,68 @@ Func _WD_CheckContext($sSession, $bReconnect = Default, $vTarget = Default)
 	Return SetError(__WD_Error($sFuncName, ($iResult) ? $_WD_ERROR_Success : $_WD_ERROR_Exception), 0, $iResult)
 EndFunc   ;==>_WD_CheckContext
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_FindElement_ByRegExp
```

Comment:
I propose that this function should be renamed for the following reasons --

- None of the existing functions utilize an underscore in their name other than the initial `_WD_`
- Other "helper" functions use the naming convention `_WD_GetElementByxxx`

Some possible alternatives --

- `_WD_GetElementByRegEx`, previously suggested without any feedback received. 
- `_WD_FindElementEx`, indicating that this function is an advanced or extended version of `_WD_FindElement`.
- `_WD_GetElementByJSSelector`

Reply:
maybe `_WD_GetElementByJSRegExp` ?

Reply:
Please take a look on: [_WD_FindElement_ByRegExp > _WD_GetElementByRegEx](https://github.com/Danp2/au3WebDriver/pull/301/commits/281028b9fbb69b81d0a99838059447a728ac09bb)

Reply:
That's fine with me. I've updated the function header in an attempt to better clarify the use of Javascript. Please review when you have a moment.